lib/pharos/config.rb

```diff
@@ -51,5 +48,25 @@ def master_host
     def worker_hosts
       @worker_hosts ||= hosts.select { |h| h.role == 'worker' }
     end
+
+    # @return [Array<Pharos::Configuration::Node>]
+    def etcd_hosts
+      etcd_hosts = hosts.select { |h| h.role == 'etcd' }
+      if etcd_hosts.empty?
+        master_hosts
+      else
+        etcd_hosts
+      end
+    end
+
+    # @return [Pharos::Configuration::Host]
+    def master_leader
+      master_hosts[0]
```

Comment:
This is the same as the `master_host`? Looks like this isn't used anywhere anymore.

Reply:
Yes, removed.